core/server/services/mega/post-email-serializer.js

```diff
@@ -169,21 +170,30 @@ const parseReplacements = (email) => {
     return replacements;
 };
 
-const getTemplateSettings = async () => {
+const getTemplateSettings = async (newsletterId = null) => {
+    let newsletter;
+    if (newsletterId) {
+        newsletter = await models.Newsletter.findOne({id: newsletterId, status: 'active'});
```

Comment:
```suggestion
        newsletter = await models.Newsletter.findOne({id: newsletterId, filter: 'status:active'});
```
I think filtering is only supported with the filter property